Styleguide/ModuleViewCompiler.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System.Collections.Concurrent;
+using System.Reflection;
+using Microsoft.AspNetCore.Mvc.ApplicationParts;
+using Microsoft.AspNetCore.Mvc.Razor.Compilation;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Primitives;
+
+namespace Forte.Styleguide
+{
+    public class ModuleViewCompiler : IViewCompiler
```

Comment:
a general comment - can we use `var` instead of concrete type wherever it's possible (in this class)

Reply:
and a question - did you write this class from scratch or it is port of some old class that existed before?

Reply:
I found this class in internet because a replacement for compiling views was necessary

Reply:
I assume this comes from here? https://stackoverflow.com/a/60901929/2393077

Can we add a comment in the code with the link to this page?